robottelo/performance/perf_stat.py

```diff
@@ -0,0 +1,41 @@
+"""Test utilities for writing csv files"""
+import csv
+import numpy as np
+
+
+def generate_stat_for_concurrent_thread(threadName, time_list,
+                                        stat_file_name, bucket_size,
+                                        num_buckets):
+    num_buckets = len(time_list)/bucket_size
+
+    # create list of bucket series
+    bucket = ['{0}-{1}'.format(bucket_size * i + 1, bucket_size * (i + 1))
+              for i in range(num_buckets)]
+
+    with open(stat_file_name, 'a') as f:
+        writer = csv.writer(f)
+        writer.writerow([])
+        writer.writerow(['{}'.format(threadName)])
+        writer.writerow([
+            'bucket',
+            'min',
+            'median',
+            'mean',
+            'max',
+            'std',
+            '90%',
+            '95%',
+            '99%'
+        ])
+        for i in range(num_buckets):
+            writer.writerow([
+                bucket[i],
+                np.amin(time_list[bucket_size*i:bucket_size*(i+1)]),
```

Comment:
Better to add a space around the math signs, like `bucket_size * i:bucket_size * (i + 1)`